model/model.go

```diff
@@ -96,6 +96,10 @@ func (c *ColumnInfo) SetDefaultValue(value interface{}) error {
 	if c.Tp == mysql.TypeBit {
 		// For mysql.TypeBit type, the default value storage format must be a string.
 		// Other value such as int must convert to string format first.
+		// The mysql.TypeBit type supports the null default value.
```

Comment:
This line is conflicted with the first sentence of this comment block. Maybe we can merge this two sentence.

Reply:
@winoros This is a special case, it is better to separate comments.